src/main.js

```diff
@@ -111,24 +114,25 @@ function createPythonFlaskProcess(serverExe) {
         env: {PATH: path.dirname(serverExe)}
       });
 
-    console.log('Started python process as PID ' + pythonServerProcess.pid);
-    console.log(serverExe)
+    logger.debug('Started python process as PID ' + pythonServerProcess.pid);
+    logger.debug(serverExe)
     pythonServerProcess.stdout.on('data', (data) => {
-      console.log(`${data}`);
+      logger.debug(`${data}`);
     });
     pythonServerProcess.stderr.on('data', (data) => {
-      console.log(`${data}`);
+      logger.debug(`${data}`);
     });
     pythonServerProcess.on('error', (err) => {
-      console.log('Process failed.');
-      console.log(err);
+      // If the python server process crashes, for now crash node also
+      logger.error(err.stack)
+      throw err
```

Comment:
Since the intent here is to crash node as well, could we add a note to the error message here about why this exception was triggered to be displayed in addition to the traceback?

And on a related note, when the python process does fail, will that process's exception be logged up on line 120 or 123 via stdout/stderr listening?

Reply:
Can confirm that the python traceback is emitted and logged via stderr listening. In fact this `.on('error' )` listener may not come into play very often.

Reply:
It looks like the 'error' listener is only used when the spawned process fails to start, or cannot be communicated with at all. I added a log message here as you suggest. 

Any excpetions from the subprocess go to the stderr listener. And in this case, with a flask process, flask is catching any uncaught python exceptions and re-raising them. So they do come through stderr, but the flask app also does not crash.